src/showcase/app/public/BUILD.bazel

```diff
@@ -28,7 +28,6 @@ ng_ts_library(
         "//src/showcase/app/public/public-examples/button-examples",
         "//src/showcase/app/public/public-examples/captcha-examples",
         "//src/showcase/app/public/public-examples/checkbox-examples",
-        "//src/showcase/app/public/public-examples/checkbox-panel-examples",
```

Comment:
Why was this removed?

Reply:
My bad, since it wasn't an example present in the business package I must've overwritten it by mistake... ⚰️